examples/demo.js

```diff
@@ -8,7 +8,26 @@ var defaultHtmlSequencerUi = require('./lib/defaultHtmlSequencerUi.js'),
 window.onload = function () {
   sequencer = ImageSequencer();
 
-  function refreshOptions() {
+  options = {
+      sortField: 'text',
+      openOnFocus: false,
+      onInitialize: function () {
+          this.$control.on("click", () => {
+            this.ignoreFocusOpen = true;
+            setTimeout(() => {
+              this.ignoreFocusOpen = false; // trigger onFocus and open dropdown
```

Comment:
Sorry but I forgot to tell you that comments need to follow certain rules from now on. We want to increase the consistency of docs in the codebase. Please start with a Capital letter and end with a full stop.

Reply:
Sure.

Reply:
Is it okey now?

Reply:
Please add full stops. Sorry for delaying this.

Reply:
Like this? Sorry I’m not sure that understood word “full stop” correctly